doc/building_picoquic.md

```diff
@@ -0,0 +1,37 @@
+# Building PicoQUIC
+
+See the 'Building Picoquic' section [README](../README.md) for building picoquic for the first time and installing the dependencies.
+
+## Dependencies
+
+- [Picotls implementation of TLS 1.3](https://github.com/h2o/picotls)
+
+
+## Picoquic as a Dependency
+
+TODO:
+_describe how to include picoquic in other projects?_
+
```

Comment:
You can find an example of that in https://github.com/Quicr/quicrq, a project prototyping media over QUIC.
This project uses picoquic as a dependency, and build picoquic if needed.

Reply:
I have started to describe `picoquic as a dependency`, but it feels like there is still something missing in order for it to be useful. I can't put my finger on it though, what's your opinion on this?